package.json

```
@@ -0,0 +1,154 @@
+{
+  "name": "@ipfs/public-gateway-checker",
```

Ensure we don't publish to NPM by mistake :))
```suggestion
  "name": "@ipfs/public-gateway-checker",
  "private": true,
```